themes/digital.gov/src/js/guide-mobile-menu.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Display horizontal desktop or vertical mobile menu when viewport size changes
+ *
+ */
+
+const dgGuideMenuBar = document.querySelector(".dg-guide__menu-bar");
+const dgAccordionMobileMenu = document.querySelector(".guide-mobile-menu");
+const dgAccordionContent = document.querySelector(
+  ".guide-mobile-menu .usa-accordion__content"
+);
+const desktopBreakpoint = window.matchMedia("(min-width: 480px)");
```

Comment:
The name `desktopBreakpoint` and its min-width being `480px`, does this mean that desktop starts at 480+?

Reply:
Yes, should it read `mobileBreakpoint` or `tabletBreakpoint` instead?